finality-aleph/src/network.rs

```diff
@@ -398,3 +543,32 @@ impl<B: BlockT + 'static, N: Network<B> + Clone> ConsensusNetwork<B, N> {
         }
     }
 }
+
+pub(crate) type RushNetworkData<B> = rush::NetworkData<Hasher, <B as BlockT>::Hash, Signature>;
+
+pub(crate) struct RushNetwork<B: BlockT> {
+    inner: GenericNetwork<RushNetworkData<B>>,
+}
```

Comment:
Why are `RushNetwork` and `GenericNetwork` separate beings?

Reply:
Mostly for testing purposes – the logic is completely independent of the data being transferred over this network, but if I were to use AlephBFT's structs directly I would have to expose ways of constructing them outside of AlephBFT.